examples/Training/java/src/training/Setup.java

```diff
@@ -97,21 +97,27 @@ private String resolveFilePath(String fileName, String directory)
 	 */
 	Setup()
 	{
-		//homeDir = "";
-		//Read the configuration file.
-		Properties p = loadConfig(resolveFilePath(CONFIG_FILE, CONFIG_DIR));
+	    String ice_config = System.getenv().get("ICE_CONFIG");
+	    Properties p;
+	    if (ice_config == null || ice_config.isEmpty()) {
+	        //fall back to training.config
+	        p = loadConfig(resolveFilePath(CONFIG_FILE, CONFIG_DIR));
+	    } else {
+	        p = loadConfig(new File(ice_config).getAbsolutePath());
+	    }
 		ConfigurationInfo info = null;
 		try {
 			info = new ConfigurationInfo();
 			info.setHostName(p.getProperty("omero.host"));
 			info.setPassword(p.getProperty("omero.pass"));
 			info.setUserName(p.getProperty("omero.user"));
-			info.setImageId(Long.parseLong(p.getProperty("omero.imageId")));
-			info.setDatasetId(Long.parseLong(p.getProperty("omero.datasetId")));
-			info.setProjectId(Long.parseLong(p.getProperty("omero.projectId")));
-			info.setPlateId(Long.parseLong(p.getProperty("omero.plateId")));
-			info.setPlateAcquisitionId(Long.parseLong(
-					p.getProperty("omero.plateAcquisitionId")));
+			info.setGroupId(Long.parseLong(p.getProperty("omero.group")));
```

Comment:
See https://ci.openmicroscopy.org/job/OMERO-5.1-merge-training/293/console, `omero.group` is currently stored as a String
```
+ java -cp .:/opt/hudson/workspace/OMERO-5.1-merge-training/OMERO.matlab/libs/omero_client.jar:/opt/hudson/workspace/OMERO-5.1-merge-training/OMERO.matlab/libs/slf4j-api.jar training/Setup
java.lang.NumberFormatException: For input string: "training_group"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Long.parseLong(Long.java:441)
	at java.lang.Long.parseLong(Long.java:483)
	at training.Setup.<init>(Setup.java:114)
	at training.Setup.main(Setup.java:145)
```